charts/enterprise-metrics/templates/graphite-write-proxy-dep.yaml

```diff
@@ -0,0 +1,136 @@
+spec:
+  replicas: {{ .Values.graphite_write_proxy.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+      release: {{ .Release.Name }}
+  strategy:
+    {{- toYaml .Values.graphite_write_proxy.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+        # The name label is important for cortex-mixin compatibility which expects certain names for services.
+        name: graphite-querier
+        gossip_ring_member: "true"
+        target: querier
+        release: {{ .Release.Name }}
+        {{- with .Values.graphite_write_proxy.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      annotations:
+{{- if .Values.useExternalConfig }}
+        checksum/config: {{ .Values.externalConfigVersion }}
+{{- else }}
+        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+{{- end}}
+        {{- with .Values.graphite_write_proxy.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ template "enterprise-metrics.serviceAccountName" . }}
+    {{- if .Values.graphite_write_proxy.priorityClassName }}
+      priorityClassName: {{ .Values.graphite_write_proxy.priorityClassName }}
+    {{- end }}
+      securityContext:
+        {{- toYaml .Values.graphite_write_proxy.securityContext | nindent 8 }}
+      initContainers:
+        {{- toYaml .Values.graphite_write_proxy.initContainers | nindent 8 }}
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - -target=graphite-write-proxy
+            - -license.path=/license/license.jwt
+            - -server.http-listen-port={{ .Values.config.server.http_listen_port }}
+            - -graphite.enabled=true
+            - -graphite.write-proxy.distributor-client.address=dns:///{{ template "enterprise-metrics.fullname" . }}-distributor.{{ .Release.Namespace }}.svc:{{ .Values.config.server.grpc_listen_port }}
+          {{- range $key, $value := .Values.graphite_write_proxy.extraArgs }}
+            - "-{{ $key }}={{ $value }}"
+          {{- end }}
+          volumeMounts:
+            {{- if .Values.graphite_write_proxy.extraVolumeMounts }}
+              {{ toYaml .Values.graphite_write_proxy.extraVolumeMounts | nindent 12}}
+            {{- end }}
+            - name: config
+              mountPath: /etc/enterprise-metrics
+            - name: runtime-config
+              mountPath: /var/enterprise-metrics
+            - name: graphite-schemas
```

Comment:
are the schemas used by the graphite write proxy?

Reply:
You're right! Addressed here https://github.com/grafana/helm-charts/pull/872/commits/48780dcc4968eb4c2a389e8ffd599eb1fcba6c58